### ISSUE TITLE
Still problems claiming GAS!

### ISSUE THREAD

**Issue by buddhaman8** [CityOfZion/neon-wallet#1211]
Please complete the form below. Any issue added that does not include these details will be closed as "not enough information".

For support using the software or concerns about the status of your funds, please go to the #support channel on the NEO Smart Economy Discord Server - https://discord.cityofzion.io

**Wallet version or commit SHA:**
e.g.: 0.2.3 or dcae88c

**Environment:**
e.g.: Win 10 Home (64 bit) Version 1709 Build 16299.309

**Prerequisites:**
e.g.: Don't have any custom tokens added before running the wallet

**Reproduction steps:**
e.g.:
1. Start Neon Wallet
2. Click on "Manage Neon settings" button
3. Click on "Manage Tokens" button
4. Click on "Add a new token" button

some description (for ex. what happens after completing reproduction steps), e.g.:
After clicking on "Add a new token" button, wallet "hangs" for 2 seconds.

**Screenshots/Video:**
e.g.: https://example.com/screenshot.png

**Comment by buddhaman8** [comment]
Still having problems claiming GAS! Keep getting transaction failed or "error 524"  - on MAC using wallet 0.2.5. This GAS issue has been going on and off for months now! We all know there are always 'bugs' to fix, but this has gone on and on for months now!

**Comment by Vinimator** [comment]
Error calculating claimable gas. Transaction Failed..!!

**Comment by MarkIsaac1** [comment]
I've suddenly got this problem as well, it was working fine the other day, i cant claim gas or send funds to Binance, i updated the wallet still no luck, i signed in using private key instead and still had the same errors Transaction Failed.. then I tried to send funds via Neo Tracker and it came up Unknown Error!

**Comment by NicksDee** [comment]
+1 same issues

**Comment by MarkIsaac1** [comment]
Is it an intentional glitch to stop people selling off their neo to put it into Binance or Tether?

**Comment by MarkIsaac1** [comment]
Could be windows as it needs an update

**Comment by NicksDee** [comment]
i didnt log to my neon wallet for 24 hrs. Later the issue was resolved all by itself. i received all NEO's transferred from Binance and all unclaimed gas.

**Comment by MarkIsaac1** [comment]
Yeah same here, i noticed the wallet or Nep 5 were down for maintenance on
Binance for a while then it sorted itself out
